metadata.rb

```diff
@@ -4,10 +4,13 @@
 name             "drupal"
 description      "Installs/Configures drupal"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "2.0.3"
+version          "2.0.4"
 recipe           "drupal", "Installs and configures Drupal"
 recipe           "drupal::cron", "Sets up the default drupal cron"
 recipe           "drupal::drush", "Installs drush - a command line shell and scripting interface for Drupal"
+depends "mysql", "~> 5.0"
```

Comment:
ahh, I missed that.  We need to peg to an explicit version as version 6 fails.  :(